src/ZoneServer/Skills/Handlers/Archer/ObliqueShot.cs

```diff
@@ -72,8 +73,16 @@ public void Handle(Skill skill, ICombatEntity caster, ICombatEntity target)
 				skillHitResult = SCR_SkillHit(caster, bounceTarget, skill);
 				bounceTarget.TakeDamage(skillHitResult.Damage, caster);
 
-				var hit = new HitInfo(caster, target, skill, skillHitResult);
-				Send.ZC_HIT_INFO(caster, bounceTarget, skill, hit);
+				var skillHit2 = new SkillHitInfo(caster, bounceTarget, skill, skillHitResult, bounceDamageDelay, skillHitDelay);
```

Comment:
The bounced shot actually uses `ZC_HIT_INFO` instead of `ZC_SKILL_FORCE_TARGET`, and it doesn't have a chance to apply slow. As such, this change might've made sense on the surface, but actually made the implementation less official.

Reply:
Based on videos, the timing of the oblique shot bounce shot is definitely after the first hit, though, so is there a way to add a damage delay to a ZC_HIT_INFO?

Also, the description seems to imply that the additional target would be slowed, I guess you've tested it on official to know that it never slows the second target?

Reply:
I have tested it, both when I initially implemented it and again yesterday to confirm it once more^^ At such an early stage it's important get official logs and behaviors and figure what there is, what you can do, and what's needed.

About `ZC_HIT_INFO`, I'm not aware of a delay option in it (yet), but I also can't confirm that the bounce shot happens after the first hit. Actually, when I test it on iToS, the bounce shot happens _first_, after the main one.

Reply:
Maybe you could post a video of this to the channel?  At any rate, if this is correct, I'll simply remove the changes to this class.

Reply:
We got a video from Kenedos.  The bounce does hit earlier, so I've reduced it's delay significantly.  It does look like it is a ZC_Hit_Info, but this causes the animation not to play, so I've left it as a force_hit for now.  Also, the bounce radius seems to be too small based on testing.  There's a value of 100 in the skill bytool, so I've used that for now.

Reply:
Heh. So I took a closer look at the skill and I know why there's no effect. You're gonna love this one. See, what they do here, instead of using `ZC_SKILL_FORCE_TARGET`, is send `ZC_HIT_INFO` and then also the `Normal` packet `PlayForceEffect`, which as you might've guessed does just that. It plays the "force" effect, or the arrow flying in this case. As such, the correct code looks something like this:
```cs
// Bounce shot
if (this.TryGetBounceTarget(caster, target, skill, out var bounceTarget))
{
	skillHitResult = SCR_SkillHit(caster, bounceTarget, skill);
	bounceTarget.TakeDamage(skillHitResult.Damage, caster);

	var hit = new HitInfo(caster, target, skill, skillHitResult);
	hit.ForceId = ForceId.GetNew();
	hit.ResultType = HitResultType.Unk8;

	Send.ZC_NORMAL.PlayForceEffect(hit.ForceId, caster, target, bounceTarget, "I_arrow009_red", 0.7f, "arrow_cast", "F_hit_good", 1, "arrow_blow", "SLOW", 800);
	Send.ZC_HIT_INFO(caster, bounceTarget, skill, hit);
}
```
(Also found that the hit's result type is different from the default.)

Reply:
Interesting, with this, rather than the second arrow being fired from the player, it's instead fired from the target, so it actually looks like it "bounces".  If you'd just let me add a delay to the sending of the bounce packet, I could even make the animation work properly.